_posts/2018-11-01-crystal-0.27.0-released.md

```diff
@@ -55,9 +55,9 @@ Again, these are preparation steps mostly, but we are shipping them already to d
 There are some missing pieces to make Crystal great for data science and more safe and sound regarding numbers. We are aiming to fix this during the next releases.
 One of the notable changes is that now `==` and `!=` will work in a sound way when comparing signed with unsigned numbers. Previously a bit comparison was used but thanks to [#6689](https://github.com/crystal-lang/crystal/pull/6689) these operators have now the right semantic. Other comparison operators were working already since [#2240](https://github.com/crystal-lang/crystal/pull/2240).
 
-In this version, a couple of arithmetic operators were added thanks to [#6890](https://github.com/crystal-lang/crystal/pull/6890): `&+`, `&-`, `&*`. They will perform additions, subtraction and multiplication with wrapping (as in, not overflowing). Some may notice that that is the exact same behaviour as `+`, `-`, `*`. In a future version, the regular operators are going to raise on overflow. This will allow you to trust the result of the operations when reaching the limits of the representable range. The ampersand operators might be useful to think of them as applying an `&& 0xFF` bit mask to the result.
+In this version, a couple of arithmetic operators were added thanks to [#6890](https://github.com/crystal-lang/crystal/pull/6890): `&+`, `&-`, `&*`. They will perform additions, subtraction and multiplication with wrapping (as in, not overflowing). Some may notice that that is the exact same behaviour as `+`, `-`, `*`. In a future version, the regular operators are going to raise on overflow. This will allow you to trust the result of the operations when reaching the limits of the representable range. The ampersand operators might be useful to think of them as applying an `& 0xFF` bit mask to the result.
 
-It will be up to you to start choosing the correct operators -- with or without ampersand. This usually depends on how the data is used: business logic and data science usually should overflow, while transport protocol implementation, checksums and low level programming might benefit from the ampersand operators.
+It will be up to you to start choosing the correct operators -- with or without ampersand. This usually depends on how the data is used: business logic and data science usually should overflow, while transport protocol implementation, checksums and low level programming might benefit from the raising operators.
```

Comment:
In transport protocols or low level, due to context the range of values could be known better. And also for efficiency one might want a plain add.

It’s up to the author for sure.

Reply:
In low-level and transport code you will be using both operators. In high-level code you will be using only overflow-checking operators. This is correct as-is.

Reply:
ok I'll revert it. And it seems I'm misunderstanding:
> business logic and data science usually should overflow

You mean it should raise on overflow, or it should overflow without error?

---
If you mean the latter I don't see the point of the examples, as you say X should overflow without error and Y might benefit from ampersand (overflow without error), repeating the same overflow policy twice.

---
If you mean the former, I think it could be written more explicitly ("should raise on overflow").

Reply:
Should raise on overflow sounds good.